lightning-client-js.js

```diff
@@ -30,20 +30,26 @@ class LightningClient extends EventEmitter {
             if (fExists(rpcPath, 'lightning-rpc')) {
             rpcPath = path.join(rpcPath, 'lightning-rpc');
             }
-            // main data directory provided, default to using the bitcoin mainnet subdirectory
-            else if (fExists(rpcPath, 'bitcoin', 'lightning-rpc')) {
-            global.logger.error(`WARN: ${rpcPath}/lightning-rpc is missing, using the bitcoin mainnet subdirectory at ${rpcPath}/bitcoin instead.`)
-            rpcPath = path.join(rpcPath, 'bitcoin', 'lightning-rpc')
+            // check subdir options in a dev -> prod order for easy development in case multiuple dirs exist on cln node
```

Comment:
Hi @bnonni this reversal of order in which the different directories are evaluated is a breaking change.

My recommendation is to add an additional evaluation of the condition for regtest and leave the order unchanged.

Reply:
Made this change too